### PR TITLE
App filter fix

### PR DESCRIPTION
**Pull request by vitordematos**

### CODE REVIEW

File: other/appfilter.xml [hunk 1]
```diff
@@ -1,4 +1,37 @@
...
+	<!-- Solunar Periods -->
+	<item component="ComponentInfo{com.forrestguice.suntimes.solunar/com.forrestguice.suntimes.solunar.MainActivit}" drawable="solunarperiods" />
+
+	<!-- Tides -->
+	<item component="ComponentInfo{com.SeventhGear.tides.TidesApplication/com.SeventhGear.tides.MainActivity}" drawable="tides" />
+	
+	<!-- Sugarizer -->
+	<item component="ComponentInfo{org.olpc_france.sugarizer/org.olpc_france.sugarizer.MainActivity}" drawable="sugarizer" />
+
+	<!-- vlc -->
```
Comment (Donnnno) [inline other/appfilter.xml:32]: I think this needs to be Speakingpal or something?

Comment (vitordematos) [reply other/appfilter.xml:32]: Please tell me if you got the updated appfilter.xml from the 11 icons I commited 2 days ago. Now I was trying to merge my two branches but maybe I commited to yours... I'm getting a little confused

Comment (Donnnno) [reply other/appfilter.xml:32]: No worries, github can be super confusing. I'll take a look.

**Comment by vitordematos** [comment]
I think maybe you've got the fix line on vlc activity (sorry, my copy+paste mistake from the external editor! Did you get it (from a merge I called "app-filter-fix")?

**Comment by Donnnno** [comment]
Yes, I've got everything that we need :) 

I'll get back to you with some feedback and changes that I've made for consistency when I'm done reviewing :^)

**Comment by vitordematos** [comment]
Thanks for the feedback and the fixing. They look great. I've used the templates but it's lack of experience with the project.

**Comment by Donnnno** [comment]
No worries, I hope it helps :^) 
(I know all this stuff can be overwhelming when you begin with it)

**Pull request merged**